source/adapters/level_zero/platform.hpp

```diff
@@ -107,4 +107,24 @@ struct ur_platform_handle_t_ : public _ur_platform {
         ze_command_list_handle_t, uint64_t, uint32_t,
         ze_event_handle_t *) = nullptr;
   } ZeMutableCmdListExt;
-};
+
+  // Structure with function pointers for External Semaphore Extension.
+  struct ZeExternalSemaphoreExtension {
+    bool Supported = false;
+    ze_result_t (*zexImportExternalSemaphoreExp)(
```

Comment:
we still use zex naming in URT?

Reply:
This is to differentiate between the L0 spec version of the symbol and the driver experimental version of the symbol.

This is the URT named version such that when we have the spec version the name can remain the same.